openwisp_radius/admin.py

```diff
@@ -617,6 +618,22 @@ class OrganizationRadiusSettingsForm(AlwaysHasChangedMixin, forms.ModelForm):
         help_text=_IDENTITY_VERIFICATION_ENABLED_HELP_TEXT,
         fallback='',
```

Comment:
@nemesisdesign shouldn't the fallback value be set to `False` here?

Reply:
@pandafy why should it be?

Reply:
You are right, this is correct.